client/human/chmain.h

```diff
@@ -1,8 +1,7 @@
 #ifndef _CHMAIN_H
 #define _CHMAIN_H
 
-#include <string>
-#include <vector>
+#include "../../util/ConfigFileProcess.h"
```

Comment:
this include is not actually needed by this file is it?  Seems to me that per the other comments from @adrianbroher  this include should just go directly into chmain.cpp.   And a question for Marcel: it seems that this chmain.h header is not actually needed (and currently does not seem to be used at all in master); the same declarations are currently simply [directly made in chmain.cpp](https://github.com/freeorion/freeorion/blob/master/client/human/chmain.cpp#L54-L55).   Seems to me we should just delete this file rather than adding it as an include to chmain.cpp (or else delete the duplicate declarations in chmain.cpp)